tests/test_smartbridge.py

```diff
@@ -309,11 +412,21 @@ async def fixture_bridge_uninit() -> AsyncGenerator[Bridge, None]:
 @pytest.fixture(name="bridge")
 async def fixture_bridge(bridge_uninit) -> AsyncGenerator[Bridge, None]:
     """Create a bridge attached to a fake reader and writer."""
-    await bridge_uninit.initialize()
+    await bridge_uninit.initialize("Caseta")
 
     yield bridge_uninit
 
 
+@pytest.fixture(name="ra3_bridge")
+async def fixture_bridge_ra3() -> AsyncGenerator[Bridge, None]:
+    """Create a RA3 bridge attached to a fake reader and writer."""
+    harness = Bridge()
```

Comment:
This code should probably use `bridge_uninit` like the Caseta version.

Reply:
ok, i've updated to match the bridge_uninit like the Caseta version